### ISSUE TITLE
Remove the blackout at the top of the screen and configure the option to hide the labels on the map.

### ISSUE THREAD

**Issue by kikuok** [organicmaps/organicmaps#950]
I would like to ask if it is possible to remove the darkening of the top of the map? It would be very helpful. Also is it possible to remove the compass and scale button? It can be distinguished as separate functions and people can turn it on at will in settings. It would be great because many people and I love the full size map without unnecessary labels.If this is not possible, can you tell me where the file with the scale and compass is so that I can delete it? Thank you. I really hope there will be such a hide feature in a future update and the top dimming will disappear.
![Uploading 125201903-cc36dd80-e279-11eb-8426-f50735ddafa4.png…]()

**Comment by biodranik** [comment]
The unusable top bar should be removed. Other elements (including the ruler) should hide/show on a tap anywhere on the map (without selecting a map feature) like it's implemented on iOS.
On iOS, we should also implement hiding the ruler and the bottom buttons bar.

**Comment by kikuok** [comment]
Yes, but when you rotate the map, the compass reappears. That is, hiding the ruler along with other elements, as well as removing the top shading will you remove in the next program updates? Thanks.

**Comment by pastk** [comment]
> Other elements (including the ruler) should hide/show on a tap anywhere on the map (without selecting a map feature) like it's implemented on iOS. On iOS, we should also implement hiding the ruler and the bottom buttons bar.

I would argue that the ruler should stay. Its meant to give a sense of the map scale. It should be considered a part of the map, not part of the UI. Just look at any printed map out there :)
